docs/testing.md

```diff
@@ -44,12 +44,81 @@ spec:
 
 ### Running the kubernetes e2e test suite
 
-The [e2e](../e2e/README.md) directory has a docker image and some scripts which make it easy to run
-the kubernetes e2e tests, using kops.
+The simple way:
+
+```
+# cd wherever you tend to put git repos
+git clone https://github.com/kubernetes/test-infra.git
+export KOPS_E2E_STATE_STORE=s3://your-kops-state-store # Change to your state store path
+export KOPS_E2E_CLUSTER_NAME=e2e.cluster.name          # Change to an FQDN for your e2e cluster name
+test-infra/jobs/ci-kubernetes-e2e-kops-aws.sh |& tee /tmp/testlog
```

Comment:
We probably need to mention that aws creds will be used and kops needs to be installed